bookings/resources.go

```diff
@@ -0,0 +1,45 @@
+package bookings
+
+import (
+	"fmt"
+	zoho "github.com/schmorrison/Zoho"
+)
+
+func (c *API) FetchResources(request interface{}, params map[string]zoho.Parameter) (data ResourceResponse, err error) {
```

Comment:
If I follow correctly, you are not providing anything to the `request` parameter (nil?) and using the `params` to provide the `resource_id` and `service_id`.

I would instead prefer to see the method signature to be the following:
```
func (c *API) FetchResources(resourceId string, serviceId string) (data ResourceResponse, err error)
```

and sending the params:
```
endpoint := zoho.Endpoint{
...
URLParameters: map[string]zoho.Parameter {
    "service_id": serviceId,
    "resource_id": resourceId,
    },
}
```

Use this pattern wherever possible.